src/entt/entity/polymorphic.hpp

```diff
@@ -0,0 +1,554 @@
+#ifndef ENTT_ENTITY_POLYMORPHIC_HPP
+#define ENTT_ENTITY_POLYMORPHIC_HPP
+
+#include "../core/type_info.hpp"
+#include "../core/type_traits.hpp"
+#include "fwd.hpp"
+#include "poly_type_traits.hpp"
+#include "poly_storage_mixin.hpp"
+
+
+/**
+ * @cond TURN_OFF_DOXYGEN
+ * Internal details not to be documented.
+ */
+
+namespace entt::internal {
+
+template<typename Convert, typename It>
+struct converting_iterator {
```

Comment:
This iterator is used in algorithms, when iterating through internal vector of `poly_pool_holder_base` inside the `poly_type` and static casting it to the typed version. However, it was used before refactoring the whole thing and now I see the much shorter solution, by just using lambda instead of iterator. So it is to be removed, and thank you for making me notice it is still there.